src/Model/Command/Sorting.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\Model\Command;
+
+/**
+ * Sort given item ids for user-based recommendations
+ */
+class Sorting extends AbstractCommand
+{
+    /** @var string */
+    private $userId;
+    /** @var string[] */
+    private $itemIds = [];
+
+    /**
+     * Sort given item ids for user-based recommendations
+     */
+    public function __construct(string $userId, array $itemIds)
+    {
+        $this->userId = $userId;
```

Comment:
In future, we may validate the values (and throw some exception on error) - like id length, format... I will create issue for that.